app/models/service_template_ansible_tower.rb

```diff
@@ -52,6 +52,10 @@ def self.validate_config_info(options)
     config_info
   end
 
+  def my_zone
+    job_template.manager.try(:my_zone) || MiqServer.my_zone
```

Comment:
I understand the `job_template.manager.try(:my_zone)` part.  Why default to `MiqServer.my_zone`?  What happens if `my_zone` just returns nil instead?